### ISSUE TITLE
Add various baseline models for comparison

### ISSUE THREAD

**Issue by jacobbieker** [openclimatefix/PVNet#162]
We want some baseline models for comparison. 

This can include:
[ ] Smart Persistence
[ ] XGBoost model

**Comment by jacobbieker** [comment]
@dfulu @Sukh-P @JackKelly what other baselines should we include?

**Comment by JackKelly** [comment]
A simple ML baseline might be nice. Perhaps just XG Boost, mapping from the NWP for a single point in space and time, to the power forecast for that point.

**Comment by Param1304** [comment]
@jacobbieker Is this issue active ? Can I work on it ?

**Comment by JackKelly** [comment]
I'm afraid that neither Jacob nor I are involved in managing the development of PVNet at the moment. So please wait for someone else from the OCF team to reply before working on this issue! Thanks!

**Comment by dfulu** [comment]
I don't think this is a priority any more. After discussing with @peterdudfield, we closed a similar issue #159 recently. I think we should close this.
